build-support/bin/release.sh

```diff
@@ -472,7 +484,7 @@ function fetch_prebuilt_wheels() {
         echo "${BINARY_BASE_URL}/${url_path}:"
         local dest="${to_dir}/${file_path}"
         mkdir -p "$(dirname "${dest}")"
-        curl --fail --progress-bar -o "${dest}" "${BINARY_BASE_URL}/${url_path}" \
+        curl --progress-bar -o "${dest}" "${BINARY_BASE_URL}/${url_path}" \
```

Comment:
This is a change because we're now passing `-SL` where we weren't before. It's a good change.